src/examples/colors/complexity.py

```diff
@@ -34,18 +37,32 @@ def generate_ib_bound(meaning, prior, ib_start = -2, ib_stop = 5, ib_step = 100)
         # else:
         # ib_boundary = ultk.rate_distortion
         ib_boundary = rd.get_ib_bound(prior=prior, meaning_dists=meaning, betas=betas)
-        #Save the IB bound to a file
+        # Save the IB bound to a file
         with open(ib_bound_filename, "wb") as f:
             pickle.dump(ib_boundary, f)
 
-    ib_boundary_points = pd.DataFrame([("ib_bound", "ib_bound", ib_point.rate, ib_point.accuracy, ib_point.distortion)
-                    for ib_point in ib_boundary if ib_point is not None], columns =['name','type','complexity', 'informativity', 'comm_cost'])
+    ib_boundary_points = pd.DataFrame(
+        [
+            (
+                "ib_bound",
+                "ib_bound",
+                ib_point.rate,
+                ib_point.accuracy,
+                ib_point.distortion,
+            )
+            for ib_point in ib_boundary
+            if ib_point is not None
+        ],
+        columns=["name", "type", "complexity", "informativity", "comm_cost"],
+    )
     ib_boundary_points.to_csv(f"{current_dir}/outputs/ib_bound.csv")
 
     return ib_boundary_points
 
 
-def generate_color_complexity(nat_langs, art_langs, meaning_dist, prior, use_rkk=False,  generate_ib_bound=False):
+def generate_color_complexity(
+    nat_langs, art_langs, meaning_dist, prior, use_rkk=False, generate_ib_bound=False
```

Comment:
Can you say more about what `meaning_dist` means?  "Meaning distribution of the color universe" is unclear to me.  Is this a distribution over color chips and their associated Meanings?  I would have thought that's the prior..., which also is not documented in the docstring